protocol/consistencychecks.go

```diff
@@ -100,6 +100,50 @@ func (cc *ConsistencyChecks) HandleResponse(requestType int, msg *Response,
 	return CheckPassed
 }
 
+// handleDirectorySTRs is supposed to be used by CONIKS clients to
+// handle auditor responses, and by CONIKS auditors to handle directory responses.
```

Comment:
Hmm, I'm not sure if this is supposed to be here since the file header states `Implements all consistency check operations done by a CONIKS client on data received from a CONIKS directory.`. Maybe we should move `verifySTR` and `verifySTRConsistency` to `str.go`? If you agree, we can add a TODO here.